src/main/resources/application-dev.yaml

```
@@ -104,4 +127,12 @@ logging:
 slack:
-  webhook-url: ${SLACK_WEBHOOK_URL}
\ No newline at end of file
+  webhook-url: ${SLACK_WEBHOOK_URL}
+
+feign:
+
+  match:
+    url: ${FEIGN_MATCH_URL}
+
+  community:
+    url: ${FEIGN_COMMNUNITY_URL}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)